parlai/crowdsourcing/tasks/turn_annotations/README.md

```diff
@@ -0,0 +1,26 @@
+# THIS TASK TRANSITION IS STILL A WIP
```

Comment:
Cool - I'll add notes about this version of the README but I'll keep in mind that it'll be revised before landing

Reply:
I'm not going to make very many changes, only to the arguments that I've changed myself. Unless you want to commandeer the PR and add the additional stuff about data saving and analysis before this lands, it likely won't change much.

Reply:
Okay, noted. If you don't think you'll have time to clean up the README, I think it's best if we leave this PR unmerged until I get a chance to revise it, so that the `parlai.crowdsourcing` READMEs are accurate :)